examples/token-airdrop-example.js

```diff
@@ -0,0 +1,419 @@
+import {
+    Client,
+    PrivateKey,
+    AccountId,
+    AccountCreateTransaction,
+    TokenAirdropTransaction,
+    Hbar,
+    TokenCreateTransaction,
+    TokenType,
+    TokenMintTransaction,
+    AccountBalanceQuery,
+    TokenClaimAirdropTransaction,
+    TokenCancelAirdropTransaction,
+    TokenRejectTransaction,
+    NftId,
+} from "@hashgraph/sdk";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+async function main() {
+    if (
+        process.env.OPERATOR_ID == null ||
+        process.env.OPERATOR_KEY == null ||
+        process.env.HEDERA_NETWORK == null
+    ) {
+        throw new Error(
+            "Environment variables OPERATOR_ID, HEDERA_NETWORK, and OPERATOR_KEY are required.",
+        );
+    }
+
+    const client = Client.forName(process.env.HEDERA_NETWORK).setOperator(
+        AccountId.fromString(process.env.OPERATOR_ID),
+        PrivateKey.fromStringDer(process.env.OPERATOR_KEY),
+    );
+
+    const CID = [
+        "QmNPCiNA3Dsu3K5FxDPMG5Q3fZRwVTg14EXA92uqEeSRXn",
+        "QmZ4dgAgt8owvnULxnKxNe8YqpavtVCXmc1Lt2XajFpJs9",
+        "QmPzY5GxevjyfMUF5vEAjtyRoigzWp47MiKAtLBduLMC1T",
+        "Qmd3kGgSrAwwSrhesYcY7K54f3qD7MDo38r7Po2dChtQx5",
+        "QmWgkKz3ozgqtnvbCLeh7EaR1H8u5Sshx3ZJzxkcrT3jbw",
+    ];
+
+    /**
+     * STEP 1:
+     * Create 4 accounts
+     */
+
+    const privateKey = PrivateKey.generateED25519();
+    const { accountId: accountId1 } = await (
```

Comment:
nit: rename the accounts: Alice, Bob, Carol, Dave, as we've discussed.

Reply:
didn't we say we are going to do that after that HIP?